components/brave_wallet_ui/common/api/hardware_keyrings.ts

```diff
@@ -7,17 +7,31 @@ import { assert } from 'chrome://resources/js/assert.m.js'
 import { BraveWallet } from '../../constants/types'
 import LedgerBridgeKeyring from '../../common/hardware/ledgerjs/eth_ledger_bridge_keyring'
 import TrezorBridgeKeyring from '../../common/hardware/trezor/trezor_bridge_keyring'
-
+import { FilecoinKeyring, LedgerKeyring } from '../hardware/hardwareKeyring'
```

Comment:
Can we name this file `components/brave_wallet_ui/common/hardware/interfaces.ts`, so we can import it like this?

```ts
import * as HWInterfaces from '../hardware/interfaces'
```

and access it like:

```
HWInterfaces.LedgerKeyring
```

so there's no confusion with the implementations of these interfaces.

Reply:
done